pypdf/_writer.py

```diff
@@ -2050,6 +2051,10 @@ def remove_objects_from_page(
             page: Page object to clean up.
             to_delete: Objects to be deleted; can be a ``ObjectDeletionFlag``
                 or a list of ObjectDeletionFlag
+            text_filters: Properties of text to be deleted. Optional.
```

Comment:
This does not render cleanly: https://pypdf--3216.org.readthedocs.build/en/3216/modules/PdfWriter.html#pypdf.PdfWriter.remove_objects_from_page